docs/backup.rst

```diff
@@ -0,0 +1,30 @@
+Downloading logs from s3
+========================
+
+1.  Install aws according to
+    `official instruction <https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html>`__.
+
+2.  Set up aws
+
+    ..  code-block:: console
+
+        $ aws configure
```

Comment:
Is it a good practice to recommend configuring the default profile? Maybe it's better to specify it:
```sh
$ aws configure --profile any_profile_name
```

Commands  below should also be changed if we decide to specify the profile.

Reply:
All of the commands should be changed. e.g.:

```
 $  aws s3 --endpoint-url https://hb.bizmrg.com --profile any_profile_name \
            ls s3://multivac/tarantool/tarantool
``` 

If we don't specify the profile, default one is used.

I don't really insist on that change. Let's either use default one, or specify it with `--profile` everywhere.

Reply:
Fixed